examples/data/stream-data.json

```diff
@@ -0,0 +1,35 @@
+[
```

Comment:
Also this schema needs to be tweaked slightly. There are keys in here that we keep for backwards compatibility sake but should be removed in this case. Namely `payload` should be removed as should any `absolute` key.